docker-compose.yml

```diff
@@ -2,6 +2,8 @@ services:
 
   registration-service:
     container_name: registration-service
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
has this been tested on Windows?

Reply:
We just tested it with Peter and it works. I updated also readme with commands how to run the system tests with powershell